exercises/common-lisp-1-a/README.md

```diff
@@ -0,0 +1,59 @@
+## Pokémon Battle
```

Comment:
To conform to how other tracks structure their readme:

```suggestion
## Instructions
```

Reply:
Done!